MAUI/ListView/viewappearance.md

```diff
@@ -0,0 +1,1208 @@
+        {
+            tappedItem.IsVisible = false;
+        }
+
+        if (tappedItem == tappedItemData)
+        {
+            tappedItem = null;
+            return;
+        }
+
+        tappedItem = tappedItemData;
+        tappedItem.IsVisible = true;
+    }
+
+    #endregion
+
+    protected override void OnDetachingFrom(ContentPage bindable)
+    {
+        listview.ItemTapped -= ListView_ItemTapped;           
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+The `IsVisible` model property which is bound to the second template will be enabled when tapping the item and disabled when tapping again the same item.
+
+## show busy indicator on list view
+
+The `SfListView` allows displaying the `SfBusyIndicator` when loading the bounded items. The busy indicator can be enabled and disabled by using `IsBusy` property.
+
+Create a `IsLoading` boolean property in view model and bind it to the `IsBusy` property. By setting the value to `IsLoading` property, the busy indicator will be enabled and disabled into the view till the items loaded in the `SfListView`.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.ListView.XForms;assembly=Syncfusion.SfListView.XForms"
+             xmlns:busyIndicator="clr-namespace:Syncfusion.SfBusyIndicator.XForms;assembly=Syncfusion.SfBusyIndicator.XForms">
+    <Grid>
+        <syncfusion:SfListView x:Name="listView" 
+                             ItemsSource="{Binding ContactInfo}" 
+                             AutoFitMode="Height">
+        </syncfusion:SfListView>
+        <busyIndicator:SfBusyIndicator x:Name="busyIndicator" InputTransparent="True"
+                                       AnimationType="SingleCircle" 
+                                       IsBusy="{Binding IsLoading, Mode=TwoWay}" 
+                                       TextColor="Magenta"
+                                       ViewBoxWidth="50"
+                                       ViewBoxHeight="50"/>
+    </Grid>
+</ContentPage>
+{% endhighlight %}
+
+{% highlight c# %}
+public class ViewModel : INotifyPropertyChanged
+{
+    private bool isLoading = false;
+
+    public bool IsLoading
+    {
+        get { return isLoading; }
+        set
+        {
+            this.isLoading = value;
+            OnPropertyChanged("IsLoading");
+        }
+    }
+
+    private async void GenerateItems()
+    {
+        IsLoading = true;
+        await Task.Delay(5000);
+        for (int i = 0; i < 30; i++)
+        {
+            var contact = new Model(CustomerNames[i], ContactNumber[i]);
+            ContactInfo.Add(contact);
+        }
+        IsLoading = false;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> When both `SfBusyIndicator` and `SfListView` loaded with same row and column, you need to set `InputTransparent` as `True` to `SfBusyIndicator` in order to pass touch interaction to `SfListView` in iOS platform.
+
+## show busy indicator on list view items
```

Comment:
Show busy indicator on list view items

Reply:
I have modified the line as per the concerns provided.